### DISCUSSION TITLE
Avoir une roadmap transverse liée aux retours de la communauté

### DISCUSSION THREAD

**Discussion by sneko** [betagouv/cap#1] [Besoin]
Solution possible : mettre en place une boîte à idées

**Comment by sneko** [comment]
C'est justement l'idée de http://cap.incubateur.net si la communauté valide 😃

**Comment by LucasCharrier** [comment]
Il y a moyen de créer ou d'enlever les catégories qui apparaissent à gauches ? (bug, avis, suggestion)

**Reply by LucasCharrier** [reply]
Je teste l'outil au passage

**Reply by sneko** [reply]
@LucasCharrier oui c'est possible pour les admins du repo (dont toi), cf. https://github.com/betagouv/cap/discussions/categories

**Reply by LucasCharrier** [reply]
Ok

**Reply by LucasCharrier** [reply]
ça à l'air bien
